documentation/projects/templates/project-proposal.md

```diff
@@ -1,4 +1,7 @@
-# Project Proposal: <!-- PROJECT NAME -->
+# Project Proposal Template
+
+```md
+# {YYYY-MM-DD} Project Proposal
```

Comment:
The changes in the existing ones put the date at the end, but the templates have it at the start. Should we make these match? Dates at the start make some sense to me as it would show the progression of the project planning chronologically when viewed alphabetically.